people/_posts/0000-03-01-dimuthu-hemachandra.md

```diff
@@ -8,15 +8,17 @@ image:
 alumni: false
 
 # social
-email: 
-github: 
+email: dhemacha@uwo.ca
+github: dimuthuhemachandra
 orcid: 
 scholar: 
-twitter:
+twitter: @dimuthuH
```

Comment:
The `@` is not needed for the twitter handle and actually causes the page not to load properly! Otherwise, lgtm